src/js/helpers/local_storage.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var default_cookies = require('./cookies');
+var local_storage_session_length = 0;
+
+module.exports = Object.assign({}, default_cookies, {
+
+  set: function(name, value, minutes, domain, excl_subdomains) {
+    // Don't break the build
+    domain = '';
+    excl_subdomains = false;
```

Comment:
Since `domain` and `excl_subdomains` are the last two parameters, we can just remove them and things should still work.

```suggestion
  set: function(name, value, minutes) {
```